packages/grid/src/mouse-handlers/index.ts

```diff
@@ -8,5 +8,5 @@ export { default as GridRowTreeMouseHandler } from './GridRowTreeMouseHandler';
 export { default as GridScrollBarCornerMouseHandler } from './GridScrollBarCornerMouseHandler';
 export { default as GridVerticalScrollBarMouseHandler } from './GridVerticalScrollBarMouseHandler';
 export { default as EditMouseHandler } from './EditMouseHandler';
-export { default as GridTokenMouseHandler } from './GridTokenMouseHandler';
+export { default as GridTooltipMouseHandler } from './GridTooltipMouseHandler';
```

Comment:
Technically this is a breaking change now, since we were exporting `GridTokenMouseHandler`. You could simply re-export `GridTooltipMouseHandler` as `GridTokenMouseHandler` in the `index.ts` so it's not a breaking change, along with a comment. Or just not bother renaming them, and just add a comment to the handler that it also helps with tooltips.
We want to avoid breaking changes when possible - we don't know if somebody is using this class or not.